localenv/cloud-nine-wallet/docker-compose.yml

```
@@ -72,7 +72,7 @@ services:
       EXCHANGE_RATES_URL: http://cloud-nine-wallet/rates
       REDIS_URL: redis://shared-redis:6379/0
       WALLET_ADDRESS_URL: ${CLOUD_NINE_WALLET_ADDRESS_URL:-https://cloud-nine-wallet-backend/.well-known/pay}
-      ILP_CONNECTOR_URL: ${CLOUD_NINE_CONNECTOR_URL}
+      ILP_CONNECTOR_URL: ${CLOUD_NINE_CONNECTOR_URL:-http://127.0.0.1:3002}
```

```suggestion
      ILP_CONNECTOR_URL: ${CLOUD_NINE_CONNECTOR_URL:-http://cloud-nine-wallet-backend:3002}
```
Same goes for happy life bank docker compose file, we should use http://happy-life-bank-backend:3002 there. 
`ILP_CONNECTOR_URL` is the one that gets communicated during auto-peering (its basically the URL the fellow peer needs to save in their DB to know where to send packets)

---

there were a few more places to make that change as well. updated here https://github.com/interledger/rafiki/pull/2784/commits/7f3d0a822b51aef27aaceda29235f1568d13aca5